app/hooks/handle_send_message.ts

```diff
@@ -201,14 +216,16 @@ export const useHandleSendMessage = ({
             };
             const accept = () => {
                 // Files are filtered on doSubmitMessage
-                sendMessage();
+                sendMessage(schedulingInfo);
```

Comment:
We probably want to do things (among others, wait for `sendMessage` when we accept, right? Or is this flow completely different?

Reply:
I'm unable to test this out right now, but I've filed a ticket (https://mattermost.atlassian.net/browse/MM-62756) for handle this.